.github/workflows/nightly.yml

```diff
@@ -0,0 +1,143 @@
+name: Nightly
+on:
+  # FIXME: Change to run on a schedule before merging
+  #schedule:
+  #  - cron: "0 1 * * *"
+  pull_request:
+    branches:
+      - '**'
```

Comment:
Consider adding a `workflow_dispatch:` trigger. This allows you to manually start the job on request if waiting for the schedule is undesired for whatever reason.